Mailbox.Tests/DataLoaderTests.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Mailbox.Tests
+{
+    [TestClass()]
+    public class DataLoaderTests
+    {
+        private List<MailBox> testMailboxes = new List<MailBox>()
+        {
+            new MailBox(Sizes.LargePremium, (1, 1), new Person("John", "Doe")),
+            new MailBox(Sizes.Small, (15, 5), new Person("Jane", "Doe")),
+            new MailBox(Sizes.Medium, (30, 10), new Person("Indiana", "Jones"))
+        };
+
+        [TestMethod()]
+        public void DataLoader_Exception()
+        {
+            //Arrange, Act & Assert
+#pragma warning disable CS8625 // Cannot convert null literal to non-nullable reference type.
+            Assert.ThrowsException<ArgumentNullException>(() => new DataLoader(null));
+#pragma warning restore CS8625 // Cannot convert null literal to non-nullable reference type.
+
+        }
+
+        [TestMethod()]
+        public void LoadTest_ExceptionReturnsNull()
+        {
+            //Arrange
+            var ms = new MemoryStream();
```

Comment:
Memory stream is `IDisposable` and should be properly cleaned up at the end of the method.
```suggestion
            using var ms = new MemoryStream();
```